Make save and open dialogs modal (#360)
NordicSemiconductor/pc-nrfconnect-programmer: Changelog.md

## 3.0.9 - UNRELEASED

### Fixed

> -   `OpenFile` and `SaveFile` dialogs are now modal to the app window

Reviewer: Suggesion:

```suggestion
-   `OpenFile` and `SaveFile` opened dialog in the background.
```
